src/runtime/virtcontainers/clh.go

```diff
@@ -692,9 +692,7 @@ func (clh *cloudHypervisor) StartVM(ctx context.Context, timeout int) error {
 	}
 	clh.state.PID = pid
 
-	// TODO: revisit the value of this timeout. It currently allows a slower guest VM startup.
-	// ctx, cancel := context.WithTimeout(context.Background(), clh.getClhAPITimeout()*time.Second)
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), clh.getClhAPITimeout()*time.Second)
```

Comment:
Let us either keep in mind that we may need to substantially increase this value as well when we receive actual SNP capability: clhAPITimeoutConfidentialGuest = 20

Or, we can adapt it now already. My suggestion would be to increment to about 40/45 seconds

Reply:
clhAPITimeoutConfidentialGuest increased from 20 to 40